### PR TITLE
Add prettier and eslint spread operator for code formatting

### PR DESCRIPTION
**Pull request by ghamaide**

### CODE REVIEW

**Comment by ghamaide** [comment]
@cRicateau @tcheymol should we update all the client/src files to match the prettier configuration ?

**Comment by ghamaide** [comment]
prettier spread

**Comment by cRicateau** [comment]
@ghamaide yes definitely, if we don't want to generate a project with tests that are failing!

**Comment by ghamaide** [comment]
@cRicateau done, can you check :)

File: generators/app/templates/client/src/modules/store.js [hunk 1]
```diff
@@ -14,21 +14,16 @@ export default function configureStore(initialState = {}, history) {
...
+  const middlewares = [sagaMiddleware, routerMiddleware(history)];
 
-  const enhancers = [
-    applyMiddleware(...middlewares),
-  ];
+  const enhancers = [applyMiddleware(...middlewares)];
 
   // If Redux DevTools Extension is installed use it, otherwise use Redux compose
-  /* eslint-disable no-underscore-dangle */
+  /* eslint-disable */
```
Comment (benja-M-1) [inline generators/app/templates/client/src/modules/store.js]: why did you disable the whole block?

Comment (ghamaide) [reply generators/app/templates/client/src/modules/store.js]: not sure, I removed it.

**Comment by ghamaide** [comment]
@cRicateau good to merge ?

**Pull request merged**